src/components/AddTodoForm.tsx

```diff
@@ -1,40 +1,37 @@
 /* eslint-disable react/prop-types */
-import { useState } from "react";
+import { FormEvent, useState } from "react";
 import InputWithLabel from "./InputWithLabel";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import style from "./AddTodoForm.module.css";
 
-const AddTodoForm = (props) => {
+const AddTodoForm = (props: { onAddTodo: any; onPostData: any }) => {
```

Comment:
in your new types.ts file add this

```
   export interface AddTodoFormProps {
        onAddTodo : (newTodo: TtodoItem) => void
        onPostData : (todoTitle: string) => Promise<TtodoItem | undefined>
    }
```

import `AddTodoFormProps` into this file 

```suggestion
    const AddTodoForm = (props: AddTodoFormProps) => {
```